src/country_data.js

```diff
@@ -681,7 +681,7 @@ const rawAllCountries = [
     'kz',
     '7',
     '+. ... ...-..-..',
-    1, ['313', '327', '7172', '312', '73622', '321', '324', '336', '318', '315', '325', '311', '326', '310']
+    1, ["310", "311", "312", "313", "315", "318", "321", "324", "325", "326", "327", "336", "7172", "73622"]
```

Comment:
Why you've replaced single quotes to double? There is only single quotes used across the file.